.github/workflows/build.yml

```diff
@@ -61,3 +66,89 @@ jobs:
         run: |
           npm run build
           git diff --name-only
+
+  pypi:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Python runtime
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+
+      - name: Set up Python dependencies
+        run: pip install --upgrade build twine
+
+      - name: Build Python package
+        run: python -m build
+
+      - name: Publish Python package
+        if: github.event_name == 'release'
+        env:
+          PYPI_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: twine upload --disable-progress-bar -u ${PYPI_USERNAME} -p ${PYPI_PASSWORD} dist/*
+
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        if: github.event_name == 'release'
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Login to GitHub Container Registry
+        if: github.event_name == 'release'
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GHCR_TOKEN }}
+
+      - name: Generate Docker tags and labels
+        id: meta
+        uses: docker/metadata-action@v4
+        with:
+          images: |
+            ${{ github.event.repository.full_name }}
+            ghcr.io/${{ github.event.repository.full_name }}
+          tags: |
+            type=ref,event=branch
+            type=ref,event=tag
+            type=ref,event=pr
+          flavor: |
+            latest=${{ github.event.release.prerelease == false }}
+
+      - name: Build Docker image
+        uses: docker/build-push-action@v3
+        with:
+          context: .
+          load: true
+          tags: ${{ steps.meta.outputs.tags }}
+          labels: ${{ steps.meta.outputs.labels }}
+
+      - name: Check Docker image
+        working-directory: /tmp
+        run: |
+          docker run --rm -i -v ${PWD}:/docs ${{ github.event.repository.full_name }}:${{ steps.meta.outputs.version }} new .
+          docker run --rm -i -v ${PWD}:/docs ${{ github.event.repository.full_name }}:${{ steps.meta.outputs.version }} build
+
+      - name: Publish Docker image
+        uses: docker/build-push-action@v3
```

Comment:
Can we upgrade this to v4? That was my initial question in #4994 😅 Otherwise dependabot will complain.

Reply:
Oh I forgot about this, let me make the change.

Reply:
Done, tested here: https://github.com/crazy-max/mkdocs-material/actions/runs/4274609950/jobs/7441347841

Reply:
Perfect! I will merge this now and then we can test-drive it pushing out 9.0.15. Really looking forward to arm64 support!

Reply:
Looks good: https://github.com/squidfunk/mkdocs-material/actions/runs/4274651507/jobs/7441421218#step:9:218